modules/advangelistsBidAdapter.js

```diff
@@ -0,0 +1,443 @@
+
+let pubid = '';
+
+export const spec = {
+  code: BIDDER_CODE,
+  supportedMediaTypes: [BANNER, VIDEO],
+
+  isBidRequestValid(bidRequest) {
+    if (typeof bidRequest != 'undefined') {
+      if (bidRequest.bidder !== BIDDER_CODE && typeof bidRequest.params === 'undefined') { return false; }
+      if (bidRequest === '' || bidRequest.params.placement === '' || bidRequest.params.pubid === '') { return false; }
+      return true;
+    } else { return false; }
+  },
+
+  buildRequests(bids, bidderRequest) {
+    let requests = [];
+    let videoBids = bids.filter(bid => isVideoBidValid(bid));
+    let bannerBids = bids.filter(bid => isBannerBidValid(bid));
+    videoBids.forEach(bid => {
+      pubid = getVideoBidParam(bid, 'pubid');
+      requests.push({
+        method: 'POST',
+        url: VIDEO_ENDPOINT + pubid,
+        data: createVideoRequestData(bid, bidderRequest),
+        bidRequest: bid
+      });
+    });
+
+    bannerBids.forEach(bid => {
+      pubid = getBannerBidParam(bid, 'pubid');
+      requests.push({
+        method: 'POST',
+        url: BANNER_ENDPOINT + pubid,
+        data: createBannerRequestData(bid, bidderRequest),
+        bidRequest: bid
+      });
+    });
+
+    console.log(requests);
+    return requests;
+  },
+
+  interpretResponse(serverResponse, {bidRequest}) {
+    let response = serverResponse.body;
+    if (response !== null && utils.isEmpty(response) == false) {
+      if (isVideoBid(bidRequest)) {
+        console.log('About to create final response for video')
+        let bidResponse = {
+          requestId: response.id,
+          bidderCode: BIDDER_CODE,
+          cpm: response.seatbid[0].bid[0].price,
+          width: response.seatbid[0].bid[0].w,
+          height: response.seatbid[0].bid[0].h,
+          ttl: response.seatbid[0].bid[0].ttl || 60,
+          creativeId: response.seatbid[0].bid[0].crid,
+          currency: response.cur,
+          mediaType: VIDEO,
+          netRevenue: true
+        }
+
+        if (response.seatbid[0].bid[0].adm) {
+          bidResponse.vastXml = response.seatbid[0].bid[0].adm;
+          bidResponse.adResponse = {
+            content: response.seatbid[0].bid[0].adm
+          };
+        } else {
+          bidResponse.vastUrl = response.seatbid[0].bid[0].nurl;
+        }
+
+        return bidResponse;
+      } else {
+        console.log('About to create final respsone for banner')
+        return {
+          requestId: response.id,
+          bidderCode: BIDDER_CODE,
+          cpm: response.seatbid[0].bid[0].price,
+          width: response.seatbid[0].bid[0].w,
+          height: response.seatbid[0].bid[0].h,
+          ad: response.seatbid[0].bid[0].adm,
+          ttl: response.seatbid[0].bid[0].ttl || 60,
+          creativeId: response.seatbid[0].bid[0].crid,
+          currency: response.cur,
+          mediaType: BANNER,
+          netRevenue: true
+        }
+      }
+    }
+  }
+/*,
```

Comment:
Is there a reason that you need to keep the commented out block? If it's for dev testing, I recommend creating a dev branch for your adapter in your Prebid.js fork

Reply:
Thanks. Have removed commented block.